nsqd/options.go

```diff
@@ -28,6 +28,12 @@ type nsqdOptions struct {
 	SyncEvery       int64         `flag:"sync-every"`
 	SyncTimeout     time.Duration `flag:"sync-timeout"`
 
+	QueueGCInterval        time.Duration
```

Comment:
Agreed on coming up with better names.  Note that I explicitly chose not to expose them as flags or config file vars (I assumed we might want to iterate and improve these knobs).